src/borg/helpers/parseformat.py

```diff
@@ -662,14 +665,26 @@ def archive(self):
 
     def get_meta(self, key, rs):
         value = self.archive.metadata.get(key, '')
-        return remove_surrogates(value) if rs else value
+        if rs:
+            if isinstance(value, list):
+                return self.format_cmdline(map(remove_surrogates, value))
```

Comment:
this assumes that every list is a command line / has to get treated like a command line.

true now, but likely not forever / in general.

Reply:
And `get_meta` defaults to an empty string if the key is not available. All bad. New attempt.